cypress/e2e/chi-custom-elements/date-picker.cy.js

```diff
@@ -281,6 +282,16 @@ describe('Date picker', function() {
       .should('have.class', `${DANGER_CLASS}`);
   });
 
+  it(`Should show a message when helper-message attribute is provided`, () => {
+    cy.get('[data-cy="test-input-error"]').as('testInputError');
+
+    cy.get('@testInputError')
+      .find('chi-date-picker').should('exist')
+      .should('have.attr', 'helper-message', `${helperMessage}`)
+      .find('chi-helper-message').should('exist')
+      .contains('div', `${helperMessage}`);
```

Comment:
Use constant for classes:

`DATE_PICKER = 'chi-date-picker';`

Can we split the test to make it more readable? I see a lot of `.should` and `.find` together and it could be confusing.

```suggestion
    cy.get('@testInputError')
      .find('DATE_PICKER')
      .should('exist')
      .and
      .should('have.attr', 'helper-message', `${helperMessage}`);
```